### ISSUE TITLE
Compile Error: unresolved reference 'NaN'

### ISSUE THREAD

**Issue by nomicode** [errata-ai/readability#4]
Using Readability 0.1.0, I am running into the following error:

``` console
$ cd .vale/styles && unzip Readability-0.1.0.zip
Archive:  Readability-0.1.0.zip
   creating: Readability/
  inflating: Readability/GunningFog.yml  
  inflating: Readability/FleschKincaid.yml  
  inflating: Readability/ColemanLiau.yml  
  inflating: Readability/AutomatedReadability.yml  
  inflating: Readability/LIX.yml     
  inflating: Readability/SMOG.yml    
  inflating: Readability/FleschReadingEase.yml  
  inflating: Readability/meta.json   
./bin/find.sh | xargs -0 vale --config .vale.ini --minAlertLevel warning --output=.vale/templates/cli.tmpl --no-wrap
E201 Invalid value provided [.vale/styles/Readability/ColemanLiau.yml:8:1]:

   7  
   8* condition: "> 9"

script run: Compile Error: unresolved reference 'NaN'
        at (main):1:37

Execution stopped with code 1.
```

**Comment by jdkato** [comment]
What version of Vale are you using?

**Comment by nomicode** [comment]
```
$ vale --version
vale version 2.15.2
```

**Comment by jdkato** [comment]
I'm seemingly unable to reproduce this. For example, using this repo,

```console
$ vale --version
vale version v2.15.2
$ cd testdata/ColemanLiau
$ vale .

 test.md
 1:1  warning  Try to keep the Coleman–Liau    Readability.ColemanLiau
               Index grade (9.58) below 9.

✖ 0 errors, 1 warning and 0 suggestions in 1 file.
```

Do you get this error on all files?

**Comment by faunaee** [comment]
I've just tried Readability, and I'm encountering similar issues.

Here's what I have:

```
$ sw_vers
ProductName:	macOS
ProductVersion:	12.2.1
BuildVersion:	21D62

$ bin/vale --version
vale version 2.15.3

$ ls -l ci/vale/Readability/
total 56
-rw-r--r--@ 1 eskwayrd  staff  249 14 Nov 17:35 AutomatedReadability.yml
-rw-r--r--@ 1 eskwayrd  staff  263 14 Nov 17:35 ColemanLiau.yml
-rw-r--r--@ 1 eskwayrd  staff  263 14 Nov 17:35 FleschKincaid.yml
-rw-r--r--@ 1 eskwayrd  staff  265 14 Nov 17:35 FleschReadingEase.yml
-rw-r--r--@ 1 eskwayrd  staff  223 14 Nov 17:35 GunningFog.yml
-rw-r--r--@ 1 eskwayrd  staff  317 14 Nov 17:35 LIX.yml
-rw-r--r--@ 1 eskwayrd  staff  212 14 Nov 17:35 SMOG.yml

$ cat ci/vale/vale.ini
StylesPath = ./
MinAlertLevel = warning # suggestion, warning or error

# Only HTML files; change to whatever you're using.
[*.{html}]

# List of styles to load.
BasedOnStyles = Fauna, Readability
```

(Those are from the Readability 0.1.0 release)

The problem(s):

```
$ bin/vale --config ci/vale/vale.ini build
E201 Invalid value provided [ci/vale/Readability/ColemanLiau.yml:8:1]:

   7  
   8* condition: "> 9"

script run: Compile Error: unresolved reference 'NaN'
	at (main):1:37

# immediately run it again
$ bin/vale --config ci/vale/vale.ini build
E201 Invalid value provided [ci/vale/Readability/GunningFog.yml:8:1]:

   7  
   8* condition: "> 10"

script run: Compile Error: unresolved reference 'NaN'
	at (main):1:37

# immediately run it again
$ bin/vale --config ci/vale/vale.ini build
E201 Invalid value provided [ci/vale/Readability/AutomatedReadability.yml:8:1]:

   7  
   8* condition: "> 8"

script run: Compile Error: unresolved reference 'NaN'
	at (main):1:37
```

It certainly appears that, on each run, the styles are loaded in a non-deterministic order, and that they all fail on the `condition` line, potentially during parsing the supplied condition string.

**Comment by jdkato** [comment]
Would it be possible to share a file that causes this?

**Comment by faunaee** [comment]
Through a divide and conquer approach, if I remove the Google site verification files from the folder that vale analyzes, the Readability styles complete successfully.

A Google site verification file has content that looks like this, including a trailing linefeed:

```
googlec75a2823d234906a.html
```

**Comment by jdkato** [comment]
Thanks for investigating.

I was able to reproduce this and it should already be fixed in v2.15.4+.
